canal/dump.go

```diff
@@ -27,6 +27,15 @@ func (h *dumpParseHandler) BinLog(name string, pos uint64) error {
 	return nil
 }
 
+func (h *dumpParseHandler) GtidSet(gtidsets string) (err error) {
```

Comment:
How about change name to SetGtidSet

Reply:
its naming style follows Binlog func, I think it's ok